### ISSUE TITLE
did you use validation set for training? 

### ISSUE THREAD

**Issue by zengyan-97** [e-bug/iglue#6]
Hi, 

Did you use validation set (in english) for training? For example, both NLVR2 and GQA have the validation set and test set. In this case, actually their test sets can be used for validation, and IGLUE for test.

**Comment by zengyan-97** [comment]
To supplement, I just want to follow your experimental settings for my experiments.

**Comment by e-bug** [comment]
Hi, we did not use the validation set in addition to the training set to train the models.

We used the English validation set for model selection during English fine-tuning.
We also used the English validation set for model selection during few-shot learning (e.g. [here](https://github.com/e-bug/iglue/blob/main/datasets/nlvr2/annotations/dev.jsonl)), except for xGQA which has human-annotated questions in multiple languages.

We also provide machine-translated annotations in the target languages for model selection during few-shot learning (e.g. [here](https://github.com/e-bug/iglue/blob/main/datasets/nlvr2/annotations/dev-zh_gmt.jsonl)), though we did not find them helpful.

**Comment by zengyan-97** [comment]
Hi, 
Thanks for your reply! I have some other questions when I am doing the experiments. 

1) solved. 

2) Table 5 reports max-shot performance. How did you implement few-shot learning? Did you fine-tune the model using the English train set first and then using few-shot samples of the target language respectively? Or, you just fine-tuned the model using few-shot samples? The latter might produce worse performance.

**Comment by e-bug** [comment]
Hi,

1. Yes, it's English test set.

2. Sorry for not making it clear in the paper (this is just mentioned in the appendix at page 22). I implemented it following your first description: I `fine-tune the model using the English train set first and then using few-shot samples of the target language respectively`. You can also see this in the experiments files (e.g. [here](https://github.com/e-bug/iglue/blob/main/experiments/few_shot/uc2/xgqa/zh/25/train.dtu.sh#L12))

**Comment by zengyan-97** [comment]
Got it! Thanks
